columnize.go

```diff
@@ -42,6 +51,22 @@ func getElementsFromLine(config *Config, line string) []interface{} {
 	return elements
 }
 
+// runeLen calculates the number of runes in a string
+func runeLen(s string) int {
```

Comment:
No. utf8.RuneLen takes one rune as an argument and returns the number of bytes used to encode this.

The runeLen function here takes a string (multiple runes) and returns how many runes are contained in it.

See https://play.golang.org/p/22NO2x_tEI